posts/intro-to-mathport.md

```diff
@@ -0,0 +1,224 @@
+  It's less interesting perhaps, but also smaller and easier to inspect.
+* https://github.com/leanprover-community/mathport contains the code for `mathport` itself,
+  as well as the continuous integration set up that runs `mathport` on Lean 3 core and `mathlib3`
+  every time there is either a PR to `mathport`, or a commit to master.
+  The artifacts produced by CI appear at https://github.com/leanprover-community/mathport/releases,
+  and the two repositories listed above have a `lakefile.lean` that will download and unpack these artifacts.
+* https://github.com/leanprover-community/mathlib4 is the current preliminary port of `mathlib` to Lean 4.
+  Later, when `mathport` begins to stabilize,
+  we will begin moving files from the output of `mathport` into this repository, but not yet.
+  For now, this repository serves several purposes:
+  * Primarily, it is the home for ports of tactics implemented in `mathlib3` to Lean 4.
+  * In order to build these tactics, a certain minimal library is necessary
+    (e.g. for the `ring` tactic, we need the notion of a `ring`!)
+    and we are constructing this by hand.
+  * There is scope for experimental developments,
+    but please don't just port parts of `mathlib3` to Lean 4
+    for the sake of doing so.
+    If you're investigating how some new Lean 4 feature might be used in the eventual port of `mathlib`,
+    it's okay to do so in this repository.
+  For now, the `mathlib4` repository has fairly low standards:
+  we don't expect the full review process used in `mathlib3`.
+  Notable parts of the `mathlib4` repository relevant for `mathport` are:
+  * `Mathlib/Mathport/SpecialNames.lean` contains a sequence of `#align` statements,
+    for cases where a definition in Lean 4 core has a different name
+    from the name that would be produced automatically by changing case conventions from Lean 4 or `mathlib3`.
+  * `Mathlib/Mathport/Syntax.lean` contains definitions of all the syntaxes of tactics currently implemented in `mathlib3`.
+    These have been written by hand by Mario, and we should take care to keep these up to date.
+    Please be careful editing this file,
+    and we also need to remember to update it if any further changes to tactics land in `mathlib3`.
+    Because in this file we only have the syntax statements,
+    if you try to use the tactics here you will get `Tactic not implemented yet` errors.
+    If you would like to work on porting tactics, essentially this file is the TODO list!
+    You should take the relevant syntax definitions,
+    move them to their own file in the `Mathlib/Tactic/` directory,
+    and provide an implementation.
+    Conversely, please do not start porting a `mathlib3` tactic without faithfully reproducing the syntax,
+    as this will then cause problems for `mathport`.
+    It's completely fine to provide an interim port, that throws errors when encountering
+    some of the bells and whistles specified by a syntax declaration.
```

Comment:
```suggestion
    * `Mathlib/Mathport/SpecialNames.lean` contains a sequence of `#align` statements,
      for cases where a definition in Lean 4 core has a different name
      from the name that would be produced automatically by changing case conventions from Lean 4 or `mathlib3`.
    * `Mathlib/Mathport/Syntax.lean` contains definitions of all the syntaxes of tactics currently implemented in `mathlib3`.
      These have been written by hand by Mario, and we should take care to keep these up to date.
      Please be careful editing this file,
      and we also need to remember to update it if any further changes to tactics land in `mathlib3`.
      Because in this file we only have the syntax statements,
      if you try to use the tactics here you will get `Tactic not implemented yet` errors.
      If you would like to work on porting tactics, essentially this file is the TODO list!
      You should take the relevant syntax definitions,
      move them to their own file in the `Mathlib/Tactic/` directory,
      and provide an implementation.
      Conversely, please do not start porting a `mathlib3` tactic without faithfully reproducing the syntax,
      as this will then cause problems for `mathport`.
      It's completely fine to provide an interim port, that throws errors when encountering
      some of the bells and whistles specified by a syntax declaration.
```